crates/wdk-build/src/metadata/driver_settings.rs

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation
+// License: MIT OR Apache-2.0
+
+//! Types for the `DriverConfig` section of the `metadata.wdk` section of the `Cargo.toml`
+//! 
+//! This section is used to specify the driver type and its associated configuration parameters.
+//! This corresponds with the settings in the `Driver Model` property pages
+
+use serde::{Deserialize, Serialize};
+
+/// The driver type with its associated configuration parameters
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash)]
+#[serde(
+    tag = "DRIVER_TYPE",
+    deny_unknown_fields,
+    rename_all = "UPPERCASE",
+    from = "DeserializableDriverConfig"
+)]
+pub enum DriverConfig {
+    /// Windows Driver Model
+    Wdm,
+    /// Kernel Mode Driver Framework
+    Kmdf(KmdfConfig),
+    /// User Mode Driver Framework
+    Umdf(UmdfConfig),
+    /// INF only drivers e.g. null drivers and extension INFs
+    Package
```

Comment:
Circling back on our earlier conversations, the WDK does use Package as a "driver type": https://github.com/microsoft/Windows-driver-samples/blob/ea026ed7e82fe5b45f0d978661e68c7f3249f3fc/general/toaster/umdf2/Package/package.VcxProj#L32C6-L32C16